src/ui/Controllers/CourseController.cs

```diff
@@ -48,7 +47,7 @@ public CourseController(ManageApi manageApi)
                     UcasCode = course.UcasCode,
                     AgeRange = courseDetail.AgeRange,
                     Route = variant.ProgramType,
-                    Qualifications= variant.ProfPostFlag,
+                    Qualifications = variant.ProfPostFlag,
```

Comment:
Is there some sort of linter we can add to catch these going forward?

Reply:
There's StyleCop but I don't like it much.